_posts/2023-08-15-OLS-joining-forces-with-communities-to-broaden-participation-in-open-science.md

```diff
@@ -0,0 +1,97 @@
+
+We will support these projects in order to champion and exemplify open science in their departments and faculties. Through local events and newsletters, we will highlight their work and invite more members to engage with their work.
+
+![Thirteen people posing for a photo, half of them sitting and hald standing, like in a primary school photo. The background is a modern building with dark greys and oranges, with copper walls. Their ages seem the go from mid twenties to their fifties](/images/2023-16-08-VU%20foto-MoniqueKooijmans-OLS-8-blog-post.jpg) Lena and the VU Amsterdam team
+
+## MetaDocencia
+Building capacity to deliver long-term cohort-based training for Spanish-speaking researchers and technicians in Latin America - Laura Acion and Nicolás Palopoli, MetaDocencia Executive Co-Directors.
+
+MetaDocencia launches its work in [two of the three NASA Transform to Open Science (TOPS)](https://www.metadocencia.org/en/post/nasatops/) mission grants by joining OLS-8 at all levels. That is, the whole MetaDocencia team, depending on where each person is, will be participating in OLS-8 as mentees, facilitators, and mentors. This participation will allow MetaDocencia and OLS teams to start collaborating in view of the two NASA projects we share. MetaDocencia will also learn OLS’ business model from the inside so we can contextualise it to our region. At the same time, participating in OLS-8 will strengthen South-South collaborations in a way our community has yet to experience. Throughout OLS-8, MetaDocencia will be working in two mentored groups. One will lead MetaDocencia’s yearly governance and policies review. The other one will build the structure and infrastructure for offering virtual cohorts to Spanish-speaking researchers and technicians in Latin America. These learnings will be shared with the broader community openly, allowing others to also learn from our experience.
+
+![The image shows 12 people of different ages, informally dressed, in a natural garden, smiling and happy.](/images/2023-16-08-metadocencia-team-OLS-8-blog-post.jpg) Part of the MetaDocencia team and external collaborators at their first in-person retreat, held in Buenos Aires in 2023. 
+
+## Making the most of this diverse cohort!
+Through this collaborative effort, we also want to reimagine some aspects of Open Seeds  so that better opportunities are made available for everyone. We would like all participants to  benefit from the diversity of expertise, perspectives and ideas members of this cohort and team will bring, and also engage with one another beyond their local contexts. In view os this goal we together thought of a couple of activities to implement in OLS-8:
```

Comment:
```suggestion
Through this collaborative effort, we also want to reimagine some aspects of Open Seeds, so that better opportunities are made available for everyone. We would like all participants to  benefit from the diversity of expertise, perspectives and ideas members of this cohort and team will bring, and also engage with one another beyond their local contexts. In view of this goal, we together thought of a couple of activities to implement in OLS-8:
```